src/deluge/util/container/array/ordered_resizeable_array.h

```diff
@@ -32,9 +32,20 @@ class OrderedResizeableArray : public ResizeableArray {
 	int32_t insertAtKey(int32_t key, bool isDefinitelyLast = false);
 	void deleteAtKey(int32_t key);
 
+#if TEST_VECTOR
+	// TODO: move to unit tests
 	void test();
-	void testSequentiality(char const* errorCode);
+#endif
+#if TEST_VECTOR_DUPLICATES
+	// TODO: move to unit tests
 	void testDuplicates();
+#endif
+#if ENABLE_SEQUENTIALITY_TESTS
+	/// \brief test that the keys in this array are sorted in ascending order.
+	///
+	/// This is used in assert-like fashion, so not for unit tests.
+	void testSequentiality(char const* errorCode);
```

Comment:
I'm not sure this is actually better, the whole function gets optimized out in the old way and it's more readable

Reply:
reverted